core/tests/puppeteer-acceptance-tests/utilities/user/topic-manager.ts

```diff
@@ -2001,6 +2001,10 @@ export class TopicManager extends BaseUser {
     await this.clickOn(' Prerequisite Skills ');
     await this.clickOn('Rubrics');
   }
+
+  async timeout(time) {
```

Comment:
Why is this function created? Is it necessary?

Reply:
nope, sorry, added by mistake (I was using that for testing)